playground/mocks/config/responseConfig.js

```diff
@@ -591,6 +591,22 @@ const passwordRecovery = {
   ],
 };
 
+const userUnlockAccount = {
+  '/idp/idx/introspect': [
+    'identify-with-unlock-account-link',
+  ],
+  '/idp/idx/unlock-account': [
+    'user-unlock-account',
+  ],
+  '/idp/idx/challenge': [
+    // 'authenticator-verification-data-phone-sms-then-voice',
+    'authenticator-verification-email'
+  ],
+  '/idp/idx/challenge/answer': [
+    'user-account-unlock-success'
+  ]
+};
+
 module.exports = {
-  mocks: idx,
+  mocks: userUnlockAccount,
```

Comment:
`nit:` If possible, can we undo this? For new engineers, this might be hard to track. If not in this commit, something to change in a different one.